source/includes/api/_dataset.md

```diff
@@ -599,7 +599,7 @@ curl -X POST https://api.resourcewatch.org/v1/dataset/find-by-ids \
 
 ## Creating a Dataset
 
-To create a dataset, you will need an authorization token. Follow the steps of this [guide](#generate-your-own-oauth-token) to get yours.
+To create a dataset, you will need an authorization token of an user with role of **MANAGER** or greater. Follow the steps of [this guide to get your token](#generate-your-own-oauth-token) or [check here if you are having problems related with authentication](#troubleshooting-auth-problems).
```

Comment:
- There's no explicit concept of role hierarchy, so "X or greater" doesn't apply. 
- Anyone can create a dataset, even USERs
- I would avoid pointing to the guide every other page, otherwise this will grow a lot with a lot of useless text. Instead we may consider giving more visibility to the troubleshooting section - 
-maybe promoting it to a top level item on the left sidebar?

Reply:
- Removed references to role hierarchy.
- Updated the text to mention it is only required to have a valid token for creating a dataset (is this correct?).
- Promoted troubleshooting section to top level section (in index.html and index-rw.html, should other docs be updated also?) and removed links to it in the permission texts.